bugsnag-android-core/src/main/java/com/bugsnag/android/Bugsnag.java

```diff
@@ -342,14 +345,14 @@ public static void leaveBreadcrumb(@NonNull String message) {
      * Leave a "breadcrumb" log message representing an action or event which
      * occurred in your app, to aid with debugging
      *
-     * @param name     A short label (max 32 chars)
+     * @param message     A short label (max 32 chars)
```

Comment:
The limit has been removed

```suggestion
     * @param message     A short label
```